puppet-client.html

```diff
@@ -239,7 +232,8 @@
             }
         },
         onLocalChange: function(patches) {
-            this.propagateDomBind(patches);
+            // todo #26
+            this.propagateDomBind({patches: patches, results: patches.map(function() { return {};})});
```

Comment:
I can't be sure if anything was removed of course, see #26. As for the overhead... the object will eventually be passed to an event, so we have to copy it somewhere (or *really* trust that no one will modify it in a event handler)